src/components/Nav/Nav.js

```diff
@@ -1,9 +1,137 @@
-import React from 'react';
-
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faMagnifyingGlass,
+  faBasketShopping,
+  faCircleUser,
+} from '@fortawesome/free-solid-svg-icons';
+import logo from './image/rush_logo.png';
 import css from './Nav.module.scss';
 
 function Nav() {
-  return <div className={css.container}>Nav</div>;
+  const [hideMenu, setHideMenu] = useState(false);
+  return (
+    <div className={css.container}>
+      <nav className={css.nav}>
+        <div className={css.nav_logo}>
+          <Link to="/">
+            <img src={logo} alt="logo" />
+          </Link>
+        </div>
+        <div className={css.nav_list}>
+          <ul className={css.nav_list_all}>
+            <li
+              className={css.nav_prd}
+              onMouseEnter={() => {
+                setHideMenu(true);
+              }}
+              onMouseLeave={() => {
+                setHideMenu(false);
+              }}
+            >
+              <Link to="">제품</Link>
+              {hideMenu ? <span className={css.selected}></span> : null}
```

Comment:
```suggestion
              {hideMenu && <span className={css.selected}></span>}
```
조건부 렌더링할 때 단축 평가를 사용하는 방법도 알고 계시면 좋을 것 같아요!

Reply:
넵~! 리뷰 감사합니다~ 👍👍👍